docs/developer_guide/self-hosted.md

```diff
@@ -168,4 +168,76 @@ Once your droplet has been created, open the console to ssh to the server, follo
 
 ![digital-ocean-console-and-ip-address](https://user-images.githubusercontent.com/8952138/141150925-9f8df004-faa0-4fbe-9875-c276d60c5118.jpg)
 
+## Backup your server 
+
+NOTE : Currently this is very hands-on implementation for the backup, please feel free to ask questions in our Discord server.
+
+### Prequisetes
+  To use backup one needs to have:
+   - Athens github repo
```

Comment:
`git clone`d Athens repository?

Reply:
yes I mean cloned athens repo, thank you.